### PR TITLE
Enabled gzip compression

### PR DESCRIPTION
**Pull request by antonkazakov**
Added gzip compression interceptor

### CODE REVIEW

**Comment by snowplowcla** [comment]
Confirmed! @antonkazakov  has signed the Individual Contributor License Agreement. Thanks so much

File: snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/tracker/interceptors/GzipInterceptorTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package com.snowplowanalytics.snowplow.tracker.interceptors;
```
Comment (BenFradet) [inline snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/tracker/interceptors/GzipInterceptorTest.java]: can you add an apache header?

Comment (antonkazakov) [reply snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/tracker/interceptors/GzipInterceptorTest.java]: sure

**Comment by antonkazakov** [comment]
Any news on these PRs?

**Comment by BenFradet** [comment]
ping @mhadam

**Comment by antonkazakov** [comment]
@mhadam Great news. Thank you!

**Comment by mhadam** [comment]
@antonkazakov Just wanted to point out that there's no collector support for gzip encoding right now, so we should make sure that's at least PR'd on a collector before this gets released.

**Comment by BenFradet** [comment]
the relevant issue is https://github.com/snowplow/snowplow/issues/4006 if you want to have a look :+1:

**Comment by AlexBenny** [comment]
This PR has been on hold for a long time because blocked by https://github.com/snowplow/stream-collector/issues/9.
Considering it seems a task with low priority, I'm closing this PR for now. We will rework on this when the relative ticket on Stream Collector is in-progress.
cc @paulboocock

**Pull request closed without merging**